src/hardware/gus.h

```diff
@@ -217,11 +217,13 @@ union ResetRegister {
 	bit_view<2, 1> are_irqs_enabled;
 };
 
+// Default state: lines disabled, latches enabled
+constexpr uint8_t MixControlRegisterDefaultState = 0b0000'1011;
```

Comment:
Looks good. It's a pity we can't initialise this via setting the named fields in the bit_view. I guess as a workaround we could just turn this into a mutable member var the initialise in the constructor. Not quite sure that's better, so probably this is fine.